lib/authlogic/session/brute_force_protection.rb

```diff
@@ -58,9 +58,9 @@ module InstanceMethods
         # By default they will be banned for 2 hours. During that 2 hour period this method will return true.
         def being_brute_force_protected?
           exceeded_failed_logins_limit? && (failed_login_ban_for <= 0 ||
-            (attempted_record.respond_to?(:last_failed_login_at) && 
```

Comment:
1. method `being_brute_force_protected?` ini masih kepanggil kah gar?
2. kenapa sampai perlu diganti pemanggilannya ke `current_login_at`?
3. ada kemungkinan efek samping?

Reply:
1. Sepertinya masih, cuman call ke `last_failed_login_at` ini ngga ada karena `exceeded_failed_logins_limit?` false. Si `exceeded_failed_logins_limit?` false karena ada hack ini https://github.com/bukalapak/mothership/blob/master/app/models/user.rb#L180
2. Simply ngerevert commit sebelumnya :sweat_smile: tagging @rolandhawk sekalian nanya kenapa ada commit ini
3. Seharusnya ngga ada karena ada hack itu

Reply:
Ini gw ubah jadi return ```false``` biar lebih clear jadinya. How2?

Reply:
LGTM kalo difalse-in 
tapi perlu tambah komen ga ya untuk future notes kenapanya (walau ini di gem?)

Reply:
Udah gw kasih komen juga diatasnya setelah commit ini, kayanya ngga keliatan di potongan kode ini